doc/en/developer/guide.md

```diff
@@ -772,54 +822,51 @@ return_type  | Int (0 or 1) | 0       | Return a hash table with one level inste
 This is an example of how to get a SNMP table:
 
 ```perl
+my $oid_rcDeviceError            = '.1.3.6.1.4.1.15004.4.2.1';
+my $oid_rcDeviceErrWatchdogReset = '.1.3.6.1.4.1.15004.4.2.1.2.0';
 
-  my $oid_rcDeviceError            = '.1.3.6.1.4.1.15004.4.2.1';
-  my $oid_rcDeviceErrWatchdogReset = '.1.3.6.1.4.1.15004.4.2.1.2.0';
-
-  my $results = $self->{snmp}->get_table(oid => $oid_rcDeviceError, start => $oid_rcDeviceErrWatchdogReset);
-
-  use Data::Dumper;
-  print Dumper($results);
+my $results = $self->{snmp}->get_table(oid => $oid_rcDeviceError, start => $oid_rcDeviceErrWatchdogReset);
 
+use Data::Dumper;
+print Dumper($results);
 ```
-
-#### get_multiple_table
+--------------
+#### 3.4 get_multiple_table
+--------------
 
 **Description**
 
-
 Return hash table of SNMP values for multiple SNMP tables.
 
 **Parameters**
 
-Parameter    | Type         | Default | Description
--------------|--------------|---------|-----------------------------------------------------------
-**oids**     | Hash table   |         | Hash table of OIDs to check (Can be set by 'load' method).
-             |              |         | Keys can be: "oid", "start", "end".
-dont_quit    | Int (0 or 1) | 0       | Don't quit even if an SNMP error occured.
-nothing_quit | Int (0 or 1) | 0       | Quit if no value is returned.
-return_type  | Int (0 or 1) | 0       | Return a hash table with one level instead of multiple.
+|Parameter    | Type         | Default | Description
+|-------------|--------------|---------|-----------------------------------------------------------|
+|oids         | Hash table   |-        | Hash table of OIDs to check (Can be set by 'load' method).|
+|-            |-             |-        | Keys can be: "oid", "start", "end".                       |
+|dont_quit    | Int (0 or 1) | 0       | Don't quit even if an SNMP error occured.                 |
+|nothing_quit | Int (0 or 1) | 0       | Quit if no value is returned.                             |
+|return_type  | Int (0 or 1) | 0       | Return a hash table with one level instead of multiple.   |      
 
 **Example**
 
 This is an example of how to get 2 SNMP tables:
 
 ```perl
+my $oid_sysDescr        = ".1.3.6.1.2.1.1.1";
+my $aix_swap_pool       = ".1.3.6.1.4.1.2.6.191.2.4.2.1";
 
-  my $oid_sysDescr        = ".1.3.6.1.2.1.1.1";
-  my $aix_swap_pool       = ".1.3.6.1.4.1.2.6.191.2.4.2.1";
-
-  my $results = $self->{snmp}->get_multiple_table(oids => [
-                                                        { oid => $aix_swap_pool, start => 1 },
-                                                        { oid => $oid_sysDescr },
-                                                  ]);
-
-  use Data::Dumper;
-  print Dumper($results);
+my $results = $self->{snmp}->get_multiple_table(oids => [
+                                                      { oid => $aix_swap_pool, start => 1 },
```

Comment:
start => 1 is not working (it's an suboid needed)

Reply:
May I ask what you suggest to replace it ? 
_This part was legacy from 2020 tutorial._